components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/signup/UserSelfRegistrationManager.java

```diff
@@ -928,10 +935,6 @@ private String getChannelVerifiedClaim(String username, String verifiedChannelTy
         // Set the verified claims to TRUE.
         setVerificationClaims(user, verificationChannel, externallyVerifiedChannelClaim, recoveryScenario, userClaims);
 
-        // Set accountState claim UNLOCK.
```

Comment:
Is this correctly set in account lock handler? Account state to unlocked?

Reply:
Yes, I verified the code and this logic handles the Account state update https://github.com/wso2-extensions/identity-event-handler-account-lock/blob/master/components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java#L607